core/storage/suggestion/gae_models.py

```diff
@@ -1116,7 +1116,7 @@ def create(
         """Creates a new TranslationContributionStatsModel instance and returns
         its ID.
         """
-        entity_id = cls.generate_id(
+        entity_id = cls.build_id(
```

Comment:
Why change this? build_id actually sounds more like a noun in some cases (it's the id of a build) so just wanted to check why the old name was not suitable. Ditto below.

EDIT: Ah, saw @vojtechjelinek's comment. Try construct_id.

Reply:
Done